codeSnippets/snippets/autoreload/build.gradle

```diff
@@ -0,0 +1,36 @@
+buildscript {
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+
+apply plugin: 'kotlin'
```

Comment:
This is the old way to apply plugins. Maybe we should use the new one.

Reply:
good point, thanks. Please check a new version